app/assets/stylesheets/topbar.scss

```diff
@@ -32,18 +50,13 @@
     height: 30px;
   }
 
-  ul.dropdown.menu li {
-    margin-bottom: 3px;
-    height: 25px;
-  }
-
-  li {
-    & a {
-    padding: 0;
-    }
+  .menu li a {
+    margin: 0px 0px 0px 0px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0px` instead of `0px 0px 0px 0px`<br>`0px` should be written without units as `0`